tests/conftest.py

```diff
@@ -7,7 +7,7 @@
 import pytest
 import xarray as xr
 from filelock import FileLock
-from xclim.indicators.land import fit, stats
+from xclim.indicators.generic import fit, stats
```

Comment:
Is this because there is deprecation warning?

Reply:
This is because we have just changed it: https://github.com/Ouranosinc/xclim/commit/0e06e2c54fc97fe0b5c67fd2802ca342622ff337

I think we need to pin this to the newest version of xclim as a result!

Reply:
Either that or we can make it backwards-compatible with a try/except block.

Reply:
Ok, let's pin to the newest xclim then.